tools/itsg33-issue-generator/script.py

```diff
@@ -0,0 +1,236 @@
+"""
+The purpose of this script is to take the CSV file with the ITSG-33 controls and create issues in github for each control.
+The script will create the issue title, body, and labels. The body will have the control definition, class, supplemental guidance,
+ references, general guide, suggested placeholder values, profile specific notes, suggested assignment, and support teams.
+"""
+
+import csv
+import requests
+import json
+import os
+import string
+import logging
+from enum import Enum
+
+"""
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+DEBUG = True/False to print debug info
```

Comment:
I like this block explaining the vars.  Can you swap the `DEBUG` one for a `LOG_LEVEL` var and default it to whatever logging level you want if it isn't specified.

Reply:
Done!